FRE-nctools_documentation_github.txt

```diff
@@ -0,0 +1,273 @@
+
+3.Cr_lake_files
+
+ Cr_lake_files tool adds GLCC Waterbod data. First reads glcc file -- get lats, lons, waterbod, pwetland and then tries averaging 0.125-degree data to 1-degree.
+ Then gets lon and lat dims from first river file should be identical for all files and opens river files 
+ read lat,lon grids, tocell, land_frac,  cellarea, and travel, then gets latitudes and longitudes of input river file.
+ Tool interpolates glcc water bodies and permanent wetlands to model grid. Finally, creates new netCDF data set; overwrite existing dataset.
+
+
+
+## Install
+
+FRE-NCtools has a collection of C and Fortran sources.  Within GFDL, FRE-NCtools
+is built using a recent version of the Intel C and Fortran compilers.  Using
+other compilers (GCC/GFortran) is possible, but is not tested by the Modeling
+Systems group at GFDL.
+
+FRE-NCtools is built using the GNU Build System.  If you received this as a
+package, the standard:
+
+```
+configure
+make
+make install
+```
+
+should be sufficient.  If you cloned the git repository, you must first run
+the `autoreconf` command with the `--install` option to copy in the missing
+autoconf files:
+
+```
+autoreconf -i
+./configure
+make
+make install
+
+
+Additionally, you will probably need to set the environment variable CONFIG_SITE
+and set the recommended environment (modules):
+
+```
+export CONFIG_SITE=/path/to/package/site-configs/<site>/config.site
+source /path/to/package/site-configs/<site>/env.sh
+```
+
+The above steps may need to be augmented depending upon your user evironment setup.
+For example, if autoreconfig is in /home/MyUsername/FRE-NCtools, it will create the
+configure command in this directory. You may tell the bash shell, for example,
+the location of both by this command:
+
+export PATH=/home/MyUsername/FRE-NCtools:$PATH
+
+Additionally, installing into a non-default directory may be desired or neccesary.
+For example, if the target directory is /home/MyUsername/bin, then FRE-NCtools
+building is configured by:
+
+autoreconf -i
+./configure --prefix=/home/MyUsername/bin
+
+Finally compile and install the tools as usual:
+
+make
+make install
+
+Some FRE-NCtools applications can be run in parallel using MPI.  If the
+`--with-mpi` option is given to `configure`, the MPI version of those
+applications will be built.
+
+A few site configurations, for GFDL managed systems, are available in the
+[site-config](site-config) directory.  These can be used by setting the
+`CONFIG_SITE` environment variable prior to running `configure`.
+
+Sources:
+
+[Geophysical Fluid Dynamics Laboratory](https://www.gfdl.noaa.gov)
+(GFDL).  These tools were primarily written by members of the GFDL
+[Modeling Systems Group](https://www.gfdl.noaa.gov/modeling-systems)
+to be used in the
+[Flexible Modeling System](https://www.gfdl.noaa.gov/fms) (FMS)
+[Runtime Environment](https://www.gfdl.noaa.gov/fre) (FRE).
+
+
+

```

Comment:
The file FRE-nctools_documentation_github.txt is not related to the fregrid tests. I don't know how that file got in there.

Reply:
@ngs333 I agree, it would be good to preserve this new more detailed documentation. I think that the additional details are suitable for inclusion in the main README.md file. If it is too much, then having a separate file would also work.

Reply:
@LuisSalbey how would you like to proceed on this additional documentation you inadvertently included in this PR? It's valuable information but is somewhat duplicated in its current form.

Option 1: Remove this .txt file entirely and create a new PR with the additional documentation

Option 2: Keep this new .txt file but remove the duplicated install details towards the end. Also include a pointer from the README.md file to this additional tool descriptions.